.rubocop.yml

```diff
@@ -38,7 +38,7 @@ Lint/AmbiguousBlockAssociation:
 
 
 AllCops:
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.2.0
```

Comment:
Nice one. Let's extract this into a separate PR so that we can merge it already. 👍

Reply:
done!